.eslintrc

```diff
@@ -15,6 +15,14 @@
     "browser": true
   },
   "rules": {
+	"no-unused-expressions": [
+	  1,
+	  {
+		"allowShortCircuit": true,
+		"allowTernary": true
+	  }
+	],
+	"no-script-url": 1,
```

Comment:
This seems to be a common one that needs to be overridden. Whenever using a onClick with a `<a>` tag and no href you need to put `href="javascript:void(0);"`.

Reply:
Yes, valid point. We are working on the eslint settings. https://github.com/salesforce/design-system-react/pull/1196